### ISSUE TITLE
Can it support RedisTimeSeries? 

### ISSUE THREAD

**Issue by zxl777** [luin/medis#289]
Or is there a plan for this.

**Comment by luin** [comment]
Yeah, it's on the roadmap. It would be helpful if you could share a bit more about what features you want to support with GUI.
